src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -18,9 +18,8 @@ const RadioButtonGroup = (props) => {
   const {
     children, name, legend, error, warning, info, onBlur,
     onChange, value, inline, legendInline, legendWidth, legendAlign,
-    legendSpacing, ml, mb, labelSpacing = 1, styleOverride
+    legendSpacing, ml, mb, labelSpacing = 1, styleOverride, isRequired
```

Comment:
Please can we use `required` instead of `isRequired` for all public components.

Reply:
no, I don't think so because required is already defined by html 5. I want to avoid this confusion.

Reply:
This is understandable, but In this case I think the developer should use `required` for HTML validation and `ariaRequired` if using formik. Either of these props should add the asterisk.

Reply:
done